front-page.php

```diff
@@ -206,6 +239,120 @@
 				</div>
 			</div>
 			<?php endif; ?>
+			<?php if($hasEvents): ?>
+			<div class="homepage__events">
+				<div class="homepage__events__background"></div>
+				<div class="row homepage__events__meta">
+					<div class="col-md-6 col-sm-12">
+						<h2 class="subheader homepage__events__subheader"><?php echo __($fieldValues->featured_events_title)?></h2>
```

Comment:
This doesn't need to be wrapped in __() function.  This should only be used for static text